justfile

```diff
@@ -12,7 +12,8 @@ generate:
 
 # run with opentelemetry
 run-telemetry:
-  OPENTELEMETRY_ENDPOINT_URL=http://127.0.0.1:55680 RUST_LOG=info,kube=debug,controller=debug cargo run --features=telemetry
+  docker-compose up -d
```

Comment:
nice. i might end up taking this out as i've killed off docker-compose deps elsewhere in favor of k3d installs, but i'll at least merge it in now for git history.